pkg/disttask/framework/handle/handle.go

```diff
@@ -267,6 +271,21 @@ func GetTargetScope() string {
 	return vardef.ServiceScope.Load()
 }
 
+// GetCloudStorageURI returns the cloud storage URI with cluster ID appended to the path.
+func GetCloudStorageURI(store kv.Storage, ctx context.Context) string {
```

Comment:
The convention is to pass the context as the first argument :)

Reply:
Yes, :(